### PR TITLE
Fix for Issue 189, resolved hstore wiht realDictCursor error

### PR DESCRIPTION
**Pull request by bgcgarlipp**
This is my proposed solution for issue #189.
It sesolves the problem where the query result is returned as a dict and not a list due to the cursor_factory being a RelDictRow

### CODE REVIEW

**Comment by bgcgarlipp** [comment]
I see it fails on this test however I am not sure why
 tests/test_connection.pytest_cancel_pending_op[9.5] ⨯

File: aiopg/connection.py [hunk 1]
```diff
@@ -39,8 +39,12 @@ def _enable_hstore(conn):
         """)
     rv0, rv1 = [], []
     for oids in (yield from cur.fetchall()):
-        rv0.append(oids[0])
-        rv1.append(oids[1])
+        if type(oids) is list:
```
Comment (jettify) [inline aiopg/connection.py]: `isinstance()` should be more robust than direct type comparisons.

**Comment by jettify** [comment]
Could you please add simple test, like create new connection with `RealDictCursor` and `enable_hstore=True` and execute simple query `select 1` or something ? This will help us not to break this feature accidentally in featue.

File: aiopg/connection.py (hunk 1 above)
Comment (bgcgarlipp) [inline aiopg/connection.py]: I'll do that

**Comment by bgcgarlipp** [comment]
I'll add a test later today or tomorrow.

**Comment by asvetlov** [comment]
ping

**Comment by jettify** [comment]
@bgcgarlipp sorry for bothering, do you have time to finish PR?

**Comment by Pliner** [comment]
Fixed by https://github.com/aio-libs/aiopg/commit/57b5d55b3593ed6c6d8507d33a8bf1e0300fce84

**Pull request closed without merging**